styles.css

```diff
@@ -87,10 +97,14 @@
 }
 
 .grid-item {
-    border: solid black 1px;
     text-align: center;
     font-size: 3.5rem;
 }
+.grid-item:hover {
+    transition: transform 100ms ease-in-out;
```

Comment:
a reasonable time duration  for motion effect of hover 🌟 Also box shadowing is a good feature to add for making the content pop out 😸 I use it myself, great work ✨